pkgs/servers/sql/postgresql/ext/omnigres.nix

```diff
@@ -0,0 +1,134 @@
+{
+  buildEnv,
+  bison,
+  boost,
+  brotli,
+  bzip2,
+  cmake,
+  doxygen,
+  fetchFromGitHub,
+  flex,
+  git,
+  lib,
+  libcap,
+  libtool,
+  libuv,
+  libxml2,
+  lld,
+  netcat,
+  openssl,
+  perl,
+  pkg-config,
+  postgresql,
+  postgresqlBuildExtension,
+  postgresqlTestExtension,
+  postgresqlPackages,
+  python3,
+  python3Packages,
+  readline,
+  stdenv,
+  zlib,
+}:
+
+let
+  # TODO: I'll remove this as well, just testing stuff
+  extensions = buildEnv {
+    name = "postgresql-custom";
+    paths = [
+      postgresql
+      postgresqlPackages.plpython3
+    ];
+  };
```

Comment:
@wolfgangwalther I want some feedback on how to proceed here.

I found out that `plpython3` and `pgcrypto` are required at build time, my idea here is to just merge these derivations and then use the whatever is inside the extensions directory.